botan-src/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "botan-src"
+version = "0.1.0+2.15.0"
+authors = ["Rodolphe Breard <rodolphe@what.tf>"]
+description = "Sources of for Botan cryptography library"
+license = "MIT"
+edition = "2018"
+homepage = "https://botan.randombit.net/"
+repository = "https://github.com/randombit/botan-rs"
+readme = "README.md"
+categories = ["cryptography"]
+
+[badges]
+travis-ci = { repository = "randombit/botan-rs" }
+
+[badges.maintenance]
+status = "actively-developed"
+
+[dependencies]
+
+[[example]]
+name = "build"
+path = "examples/build.rs"
```

Comment:
Ideally there would also be a simple test that just verifies that something links, eg invoke `botan_ffi_api_version` and verify it returns a positive integer.